QUICKSTART: prefer resolve:ssm: parameters for simplicity

tjkirch opened the pull request:
```
The "simple option" was originally the note at the end of this section about
using the SSM web console.  That link changed, but we also discovered that you
have to have created a parameter in that region yourself to be able to use the
search quicklink.  Requiring that would be awkward, so we remove the link.

A simpler option for most people is to use the resolve:ssm: prefix to have EC2
dynamically fetch the latest AMI ID from SSM.  This updates the wording to
prefer that option, creating a new subsection for manually fetching parameters.
```

I also didn't think there was a reason to have the wording at the top of the ECS quickstart be different from the k8s one, so I made those match as part of the change.

**Testing done:**

Ran launches with the given `resolve:ssm:` prefixes to confirm they're correct.  Ran the updated aws-cli commands to ensure they still gave our results.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

tjkirch committed:
QUICKSTART: prefer resolve:ssm: parameters for simplicity

The "simple option" was originally the note at the end of this section about
using the SSM web console.  That link changed, but we also discovered that you
have to have created a parameter in that region yourself to be able to use the
search quicklink.  Requiring that would be awkward, so we remove the link.

A simpler option for most people is to use the resolve:ssm: prefix to have EC2
dynamically fetch the latest AMI ID from SSM.  This updates the wording to
prefer that option, creating a new subsection for manually fetching parameters.

samuelkarp commented on QUICKSTART-ECS.md:
```diff
@@ -36,38 +36,63 @@ aws ecs --region us-west-2 create-cluster --cluster-name bottlerocket
 
-You can either use an official AMI provided by Amazon or build an AMI yourself using
-[our guide](https://github.com/bottlerocket-os/bottlerocket/blob/develop/BUILDING.md).
+You can either build an AMI yourself using [our guide](BUILDING.md), or use an official AMI provided by Amazon.
```
For a quickstart, I think pointing to the official AMIs first ends up being a better experience.  I prefer the language that was already here (and maybe we make the EKS one match).

tjkirch replied:
I'm thinking it might be best to remove the BUILDING link here altogether.  This is a quickstart for people to launch Bottlerocket.  The README has links to BUILDING right after the links to the quickstarts for people who have more advanced uses.

samuelkarp replied:
That works for me too.

tjkirch replied:
Added a commit with that simplification.

tjkirch committed:
QUICKSTART: Remove BUILDING link

This was put in for completeness, but goes against the purpose of the
quickstart guides, which is to get an interested user launching Bottlerocket
quickly, and it's awkward to word such a different purpose here.

The BUILDING guide is linked from README right after the links to the
quickstart guides, for users that have more advanced needs and need to build
their own images.
